python/README.md

```diff
@@ -15,12 +15,6 @@ Install the latest release of `ccao` from PyPI:
 pip install ccao
 ```
 
-You can also install the most recent code directly from GitHub:
-
-```bash
-pip install "git+https://github.com/ccao-data/ccao.git#egg=ccao&subdirectory=python"
-```
```

Comment:
This no longer works now that we're removing the data file symlink, so I'm removing the instructions. Expand the diff above to see the instructions for installing from PyPI (which are currently not technically correct, but this PR will remedy that).